cookbooks/aws-parallelcluster-config/spec/unit/resources/manage_raid_mount_unmount_spec.rb

```diff
@@ -0,0 +1,196 @@
+require 'spec_helper'
+
+class Chef::Resource::RubyBlock
+  def wait_for_block_dev(path)
+    # do nothing
+  end
+end
+
+def mock_get_raid_devices(devices)
+  allow_any_instance_of(Chef::Resource).to receive(:get_raid_devices).with("/dev/md0").and_return(devices)
+end
+
+describe 'manage_raid:mount' do
+  for_all_oses do |platform, version|
+    context "on #{platform}#{version}" do
+      cached(:venv_path) { 'venv' }
+      cached(:raid_superblock_version) do
+        if platform == 'redhat' || "#{platform}#{version}" == 'ubuntu20.04'
```

Comment:
Nitpick. You can use ternary operator: `platform == 'redhat' || "#{platform}#{version}" == 'ubuntu20.04' ? '1.2': '0.90'`